eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -41,15 +41,15 @@ jobs:
 
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'Linux_musl_x64') }}:
+      - (Alpine.314.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64-20210910135833-1848e19
       - ${{ if eq(parameters.jobParameters.isFullMatrix, false) }}:
-        - (Alpine.312.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200602002622-e06dc59
-      - ${{ if eq(parameters.jobParameters.isFullMatrix, true) }}:
-        - (Alpine.312.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200601195603-e06dc59
+        - (Alpine.313.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-helix-amd64-20210910135845-8a6f4f3
```

Comment:
Do we want to keep running 313 on PRs? -- that means on PRs we will run 313 and 314? Maybe we should keep 313 as rolling only? 

cc: @jkotas @aik-jahoda @ViktorHofer

Reply:
Agreed. Should 3.13 vs. 3.14 (latest; released 3 months ago) follow the same logic which @aik-jahoda used for Ubuntu (https://github.com/dotnet/runtime/pull/59578)?

Reply:
We should only have one Alpine version on PRs.

Reply:
In my opinion running latest on PRs makes more sense.

Reply:
Updated with 314-only queues.